src/components/views/messages/MessageActionBar.js

```diff
@@ -114,13 +114,19 @@ export default class MessageActionBar extends React.PureComponent {
     static contextType = RoomContext;
 
     componentDidMount() {
-        this.props.mxEvent.on("Event.decrypted", this.onDecrypted);
+        if (this.props.mxEvent.isBeingDecrypted()) {
+            this.props.mxEvent.once("Event.decrypted", this.onDecrypted);
+        }
         this.props.mxEvent.on("Event.beforeRedaction", this.onBeforeRedaction);
+        if (this.props.mxEvent.getId()[0] !== "!") {
```

Comment:
Can we expressing this as a function with an easy to read name? Does `isSending` work?

```suggestion
        if (this.props.mxEvent.isSending()) {
```